CONTRIBUTING.md

```diff
@@ -251,7 +251,9 @@ When declaring a record in XML:
 * Try to group the records by model. In case of dependencies between
   action/menu/views, the convention may not be applicable.
 * Use naming convention defined at the next point
-* The tag `<data>` is only used to set not-updatable data with `noupdate=1`
+* The tag `<data>` is only used to set not-updatable data with `noupdate=1` when your data file contains a mix of "noupdate" data, otherwise, you should use one of this:
```

Comment:
I think this should be two sentences: `data. Otherwise,`

Also:
 s/one of this/one of these